lib/effectiveCallNumber.js

```diff
@@ -14,21 +14,13 @@ import { get } from 'lodash';
  *   <EffectiveCopy>
  *
  * @param {object} item an item record as returned from /inventory/items/${item-id}
- * @param {object} holdingsRecord an optional holdings record returned from /holdings-storage/holdings/${holdingsrecord-id}
  * @return {string} the effective call number
  */
-export default function effectiveCallNumber(item, holdingsRecord) {
-  const prefix = get(item, 'callNumberComponents.prefix') ||
-    get(holdingsRecord, 'callNumberPrefix', '');
-  const callNumber = get(item, 'callNumberComponents.callNumber') ||
-    get(holdingsRecord, 'callNumber', '');
-  const suffix = get(item, 'callNumberComponents.suffix') ||
-    get(holdingsRecord, 'callNumberSuffix', '');
-
+export default function effectiveCallNumber(item) {
```

Comment:
If this function is intended to be used in both inventory and circulation, then it needs to understand both `effectiveCallNumberComponents` and `callNumberComponents` properties.

This change appears to swap it to the inventory item object name. Will this break the functionality in circulation?

Reply:
@marcjohnson-kint thanks I just adjusted it a bit more.